src/object/cli_obj.c

```diff
@@ -1718,15 +1718,20 @@ dc_obj_retry_delay(tse_task_t *task, int err, uint16_t *retry_cnt, uint16_t *inp
 		   uint32_t timeout_sec)
 {
 	uint32_t	delay = 0;
+	uint32_t	limit = 4;
 
 	/*
-	 * Randomly delay 5 - 68 us if it is not the first retry for
+	 * Randomly delay 5 ~ 1028 us if it is not the first retry for
 	 * -DER_INPROGRESS || -DER_UPDATE_AGAIN cases.
```

Comment:
I think RPC should use millisecond instead of microsecond?

Reply:
This is general logic for RPC retry because of non-committed DTX, that usually happens for fetch case. The next resent RPC maybe be sent to related DTX leader on which the DTX maybe committable. If we use millisecond, the delay maybe too large.